llrt_core/src/vm.rs

```diff
@@ -583,14 +585,24 @@ fn init(ctx: &Ctx<'_>, module_names: HashSet<&'static str>) -> Result<()> {
             };
             let import_name = if module_names.contains(specifier.as_str())
                 || BYTECODE_CACHE.contains_key(&specifier)
-                || specifier.starts_with('/')
+                || Path::new(&specifier).is_absolute()
```

Comment:
We have a special `is_absolute` method in llrt path module. This is a bit more optimized than Path from rust

Reply:
Hi @richarddavison , We have corrected the problem, please review :)